wp-content/plugins/zippicks-geo-service/includes/class-geohash.php

```
@@ -0,0 +1,281 @@
+     */
+    public function decode($geohash) {
+        $lat_range = [-90.0, 90.0];
+        $lng_range = [-180.0, 180.0];
+        $is_lng = true;
+        
+        for ($i = 0; $i < strlen($geohash); $i++) {
+            $cd = strpos($this->base32, $geohash[$i]);
+            
+            for ($mask = 16; $mask > 0; $mask >>= 1) {
+                if ($is_lng) {
+                    $lng_err = ($lng_range[1] - $lng_range[0]) / 2;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$lng_err'. (Unused Code Rules)

(UnusedLocalVariable)